Extensions/ParticleSystem/particleemitterobject-pixi-renderer.js

```diff
@@ -0,0 +1,264 @@
+
+
+
+gdjs.ParticleEmitterObjectPixiRenderer = function(runtimeScene, runtimeObject, objectData){
+    var texture = null;
+    var graphics = new PIXI.Graphics();
+    graphics.lineStyle(0, 0, 0);
+    graphics.beginFill(gdjs.rgbToHexNumber(255,255,255), 1);
+    if(objectData.rendererType === "Point")
+        graphics.drawCircle(0, 0, objectData.rendererParam1);
+    else if(objectData.rendererType === "Line")
+        graphics.drawRect(0, 0, objectData.rendererParam1, objectData.rendererParam2);
+    else{
+        if(objectData.textureParticleName){
+            var sprite = new PIXI.Sprite(runtimeScene.getGame().getImageManager().getPIXITexture(objectData.textureParticleName));
+            sprite.width = objectData.rendererParam1;
+            sprite.height = objectData.rendererParam2;
+            graphics.addChild(sprite);
+        }
+        else{
+            graphics.drawRect(0, 0, objectData.rendererParam1, objectData.rendererParam2);
+        }
+    }
+    graphics.endFill();
+    texture = graphics.generateTexture();
+
+    var config = {
+        color: {
+            list: [
+                {
+                    value: gdjs.rgbToHexNumber(objectData.particleRed1,
+                                               objectData.particleGreen1,
+                                               objectData.particleBlue1).toString(16),
+                    time: 0
+                },
+                {
+                    value: gdjs.rgbToHexNumber(objectData.particleRed2,
+                                               objectData.particleGreen2,
+                                               objectData.particleBlue2).toString(16),
+                    time: 1
+                }
+            ],
+            isStepped: false
+        },
+        speed: {
+            list: [
+                {
+                    value: objectData.emitterForceMin,
+                    time: 0
+                },
+                {
+                    value: objectData.emitterForceMax,
+                    time: 1
+                }
+            ],
+            isStepped: false
+        },
+        acceleration: {
+            x: objectData.particleGravityX,
+            y: objectData.particleGravityY
+        },
+        lifetime: {
+            min: objectData.particleLifeTimeMin,
+            max: objectData.particleLifeTimeMax
+        },
+        frequency: 1.0/objectData.flow,
+        spawnChance: 1,
+        particlesPerWave: 1,
+        maxParticles: objectData.maxParticleNb,
+        emitterLifetime: -1,
+        pos: {
+            x: 0,
+            y: 0
+        },
+        addAtBack: false,
+        spawnType: "circle",
+        spawnCircle: {
+            x: 0,
+            y: 0,
+            r: objectData.zoneRadius
+        }
+    };
+
+    // Small fix because different uses, instead random min and max speeds,
```

Comment:
Good idea to add a comment here, however I think you can rephrase it: 

Small fix because different uses => We need to adapt a bit the configuration of the speed of particles
instead random min and max speeds => instead of random minimum and maximum speed
it uses initial and final speed => pixi-particles uses initial and final speed

For such code that is a bit obscure without a comment, it's nice to have a comment that is as descriptive as possible and without ambiguity (what is `it`? ;))